### PR TITLE
Remove the Bulk Edit capability from comments management.

### PR DESCRIPTION
**Pull request by kwight**
This PR temporarily removes the Bulk Edit button from comments navigation – it's been moved to a later milestone, while we concentrate on releasing M1.

screen shot 2017-07-17 at 2 58 26 pm

**Testing**
* Load a valid `/comments` route and be sure that you don't have a button for enabling bulk actions.

### CODE REVIEW

**Comment by Copons** [comment]
What if, instead of removing this, we hid this behind a `manage/comments/bulk-actions` feature flag?
I'm sure it will turn out useful for m2 development anyway.

File: client/my-sites/comments/comment-navigation/index.jsx [hunk 1]
```diff
@@ -172,12 +172,6 @@ export class CommentNavigation extends Component {
 					) }
 				</NavTabs>
 
-				<CommentNavigationTab className="comment-navigation__actions comment-navigation__open-bulk">
```
Comment (gwwar) [inline client/my-sites/comments/comment-navigation/index.jsx]: Another option is to do what we did below with search, and add a boolean prop: useBulkActions, that defaults to false. We can remove the prop when we want to show it always.

Comment (Copons) [inline client/my-sites/comments/comment-navigation/index.jsx]: True, but a feature flag will come in handy if we want to develop the bulk actions in smaller steps while the m1 is live.

Comment (gwwar) [inline client/my-sites/comments/comment-navigation/index.jsx]: Haha, either way. We'll be working on this in the short term so I'm flexible on options here.

**Comment by kwight** [comment]
> What if, instead of removing this, we hid this behind a manage/comments/bulk-actions feature flag?

Oooooh, yeah, I like that – done in 0845b21.

**Comment by kwight** [comment]
> Optional, but looks like we're only using isEnabled

Oh, well spotted – fixed in b02d1197a83927f31c78096e24fe53841f287fef.

**Comment by kwight** [comment]
> If left undefined, new features default to false, but I do like adding entries to each config. It's optional here since we'll likely be removing this soon.

Yes, since it's planned for the next milestone, I left it out of all the other environments.

**Pull request merged**